spec/models/topic_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
```

Comment:
This model spec is kind of pointless.

The rule of testing is that you test your own code, not the libraries and other dependencies that your code is dependent on. In this test you're testing that you can save a topic, which, because it inherits saving functionality from ActiveModel/Record, means you're just testing ActiveModel/ActiveRecord.

I'd delete this spec file entirely at this point.

Sandy Metz has a great talk on testing (https://www.youtube.com/watch?v=URSWYvyc42M), which covers some of the other dos and don'ts of testing